lib/workers/repository/process/index.ts

```diff
@@ -130,6 +130,10 @@ export async function extractDependencies(
     const extracted: Record<string, Record<string, PackageFile[]>> = {};
     for (const baseBranch of config.baseBranches) {
       addMeta({ baseBranch });
+
+      if (scm.syncForkWithUpstream) {
+        await scm.syncForkWithUpstream(baseBranch);
+      }
```

Comment:
We could also check `forkToken` here, so we do not to call `git.getRemotes` & fetch usptream each time when we are processing non-fork cases. 
```suggestion

      if (config.forkToken && scm.syncForkWithUpstream) {
        await scm.syncForkWithUpstream(baseBranch);
      }
```